Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocAutoCompleteReferenceValue.UI.css

```diff
@@ -98,6 +98,14 @@
   background-repeat: no-repeat;
 }
 
+.ac_placeholder_loading > div:first-child
+{
+  background-image: url('../Image/sprite.svg#AutoCompleteReferenceValueRequestInProgressIndicator');
```

Comment:
do you think we should de-duplciate the url into a css variable local to the commen ancestor of ac_loading and ac_placeholder_loading?